### ISSUE TITLE
MoveCraft will not detect Train...

### ISSUE THREAD

**Issue by ColeSkory** [msummers123/Movecraft-3#168]
Hello there! We are trying to make our Hogwarts express train become a controllable vehicle. We had someone who really knew Movecraft make is a train.craft file but whenever we try to control the train it either says tells us that a block that is part of the train is in the way, or it's too small/large.

Here is the .craft file

name: Train
maxSize: 20000
minSize: 10
allowedBlocks: 
    - 159:14
    - 85
    - 173
    - 192
    - 171:0
    - 164
    - 102
    - 159:9
    - 160:15
    - 169
    - 41
    - 44:1
    - 35:8
    - 35:7
    - 35
    - 42
    - 77
    - 76
    - 109
    - 89
    - 323:0

forbiddenBlocks:
    - 7
    - 7
    - 160:8
    - 171:7
    - 98
    - 44:5
    - 1:5
    - 139

canCruise: true
canFly: true
maxHeightLimit: 200
maxHeightAboveGround: 100
vertCruiseSkipBlocks: 1
collisionExplosion: 0.0
speed: 5.0
cruiseSkipBlocks: 1
cruiseSpeed: 6.0
fuelBurnRate: 0.0
sinkPercent: 99.0
overallSinkPercent: 60.0
tryNudge: false
canStaticMove: true
maxStaticMove: 2
flyblocks:
    35: #wool represents lift
        - 0.0
        - 100.0

**Comment by keough99** [comment]
For the block thing with multiple ids to it like the 171:2 for example for those from what I know you need to put "171:2" I think that's right

**Comment by ColeSkory** [comment]
So you're saying to use air quotes?

**Comment by keough99** [comment]
Yes this is what I remember seeing in some of my craft types check some of the stock craft types that are pre made it should show the proper way

**Comment by ColeSkory** [comment]
Dude! You are a life saver! THANK YOU! I have one more problem though.... It is saying there is a forbidden block on the train... Do you have a Skype? Mine is PorkRoasterBoy

**Comment by ColeSkory** [comment]
Ok, I figured it out. But now the sign won't stick to the train....

**Comment by ColeSkory** [comment]
And there's another thing... I can't seem to turn the train...

**Comment by keough99** [comment]
I sent u a skype request I can help as much as I can
